dapp/src/components/WalletSelectContent.js

```diff
@@ -107,8 +114,19 @@ const WalletSelectContent = ({}) => {
           return (
             <button
               key={name}
-              className="connector-button d-flex align-items-center"
-              onClick={() => onConnect(name)}
+              className={`connector-button d-flex align-items-center ${
+                isMobile && name === 'Ledger' ? 'grey' : ''
+              }`}
+              onClick={() => {
+                if (isMobile && name === 'MetaMask') {
+                  setError(null)
+                  window.location.href = process.env.METAMASK_DEEPLINK
```

Comment:
staging / prod / local environment should have this env var available

Reply:
yep, forgot to add